rust-src/ecvrf/src/public.rs

```diff
@@ -1,18 +1,16 @@
 //! ed25519 public keys.
 
+use anyhow::Result;
```

Comment:
Don't import anyhow::Result, either use ParseResult, rename it, or use it qualified.